www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -40,16 +53,38 @@ import {
 
 const useStyles = makeStyles((theme) => ({
   filterBox: {
-    padding: theme.spacing(2),
+    padding: theme.spacing(1),
     backgroundColor: theme.palette.common.white,
   },
+  filterLineLabel: {
+    width: 60,
+    textAlign: 'center',
+  },
   filterGroup: {
-    minWidth: 250,
+    minWidth: 200,
+  },
+  searchField: {
+    width: 500,
   },
   autocompleteField: {
-    minWidth: 250,
+    minWidth: 200,
     maxWidth: 400,
   },
+  expandButton: {
+    float: 'right',
+    transform: 'rotate(0deg)',
+    marginLeft: 'auto',
+    padding: theme.spacing(1),
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpenButton: {
+    transform: 'rotate(180deg)',
+  },
```

Comment:
I think we can live without the animation for now

Reply:
Do you want I set transition duration to 0 ?

Reply:
Done. I used ExpansionPanel component from material-ui